tests/commands/build.spec.ts

```diff
@@ -115,6 +116,85 @@ describe('balena build', function() {
 		});
 	});
 
+	it('should create the expected tar stream (--emulated)', async () => {
+		const projectPath = path.join(projectsPath, 'no-docker-compose', 'basic');
+		const transposedDockerfile =
+			stripIndent`
+			FROM busybox
+			COPY [".balena/qemu-execve","/tmp/qemu-execve"]
+			COPY ["./src","/usr/src/"]
+			RUN ["/tmp/qemu-execve","-execve","/bin/sh","-c","chmod a+x /usr/src/*.sh"]
+			CMD ["/usr/src/start.sh"]` + '\n';
```

Comment:
This test case checks that the tar stream produced by `balena build --emulated` contains a duly transposed Dockerfile that prefixes relevant commands with `"/tmp/qemu-execve"`. (Like all other test cases in this file, the docker daemon is mocked and no image is actually built.)